File: lab07/count_word.py
#!/usr/bin/python
import sys
import re
argList = sys.argv
wordToMatch = argList[1];
total = 0

wordToMatch = wordToMatch.lower()
for line in sys.stdin:
    words = re.split(r"[^a-zA-Z]", line)
    for word in words:
        word = word.lower()
        if word == wordToMatch:
            total += 1

print ("%s occurred %d times" % (wordToMatch, total))

File: lab07/count_word.pl
#!/usr/bin/perl -w

$wordToSearch = lc($ARGV[0]); # get the word from the commandline
$total = 0;
while ($line = <STDIN>){
	@words = split /[^a-zA-Z]/, $line;
	foreach $word (@words){
		if (lc($word) eq $wordToSearch){ # the word was found
			$total++;
		}
	}
}

print "$wordToSearch occurred $total times\n";
